MarkdownAsset.js

```diff
@@ -12,27 +12,59 @@ class MarkdownAsset extends Asset {
       linkify: true,
       typographer: true
     }).use(Meta)
-    // try loading 'markdown-it-highlight' if available
+    // try loading 'markdown-it-highlightjs' if available
     try {
-      md = md.use(require('markdown-it-highlight').default)
-    } catch (e) { }
+      md = md.use(require('markdown-it-highlightjs'), {})
+    } catch (e) {
+      if (e.code !== 'MODULE_NOT_FOUND') {
+        throw e;
+      }
+    }
     this.md = md
   }
 
+  collectDependencies () {
+    this.collectImgs(this.ast.parsed)
+  }
+
+  collectImgs (tokens) {
+    for (const token of tokens) {
+      if (token.type === 'inline' && token.children !== null && token.children.length > 0) {
+        this.collectImgs(token.children)
+      } else if (token.type === 'image') {
+        const attrs = token.attrs.map((attr) => {
+          const [name, value] = attr;
+          if (name === 'src' && value.startsWith('.')) {
```

Comment:
if the image is not under src folder, then it won't be processed, am I correct? why do we want to enforce images to be under `src` forlder? I'd just remove the `name === 'src'`

Reply:
The `name` and `value` here is the name and value of the attribute. It checks the value of attributes named "src".
So it keeps the same logic as in your original PR, just written differently, since I needed to update the set of attributes.

Reply:
Got it!